Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -215,5 +224,104 @@ public String visualizeAllNetworks(@PathVariable("projectUnixName") String proje
+                return (m2.getValue()).compareTo(m1.getValue());
+            }
+        });
+        
+        JSONArray obj = new JSONArray();
+       
+        for (int k = 0; k< (list.size()>10?10:list.size()); k++){
+            Map.Entry<String, Integer> entry = list.get(k);
+            String[] temp = map2.get(entry.getKey());
+            JSONObject jo = new JSONObject();
+            jo.put("conceptId",entry.getKey());
+            jo.put("description", temp[0]);
+            jo.put("label", temp[1]);
+            jo.put("frequency", entry.getValue());
+            obj.put(jo);
+        }
+        return obj;
+    }
+	
+	/**
+	 * This method gives the visualization of  how often concepts appear in the networks
+	 * @author Bharath Srikantan & Ajay Modi
+	 * @param projectUnixName	The project unix name
+	 * @param model				Model
+	 * @return view
+	 * @throws JAXBException
+	 * @throws QuadrigaStorageException
+	 */
+	@RequestMapping(value = "sites/{projectUnixName}/statistics", method = RequestMethod.GET)
+	public String showProjectStatistics(@PathVariable("projectUnixName") String projectUnixName,
+									   Model model)throws JAXBException, QuadrigaStorageException {
+		IProject project = getProjectDetails(projectUnixName);
+
+		if (project == null) {
+			return "auth/accessissue";
+		}
+		
+		
+        String projectid = project.getProjectId();
+        List<INetwork> Networks = networkmanager.getNetworksInProject(projectid);
+        HashMap<String,Integer> top10Concepts = new HashMap<String, Integer>();
+        HashMap<String,String[]> top10ConceptsDetails = new HashMap<String, String[]>();
+        JSONArray jo = new JSONArray();
+        StringBuffer errorMsg = new StringBuffer();
+        StringBuffer successMsg = new StringBuffer();
+        //List<String> networkNames = null;
+        if(!Networks.isEmpty()){
+            
+            for (int i = 0; i < Networks.size(); i++) {
+                Network n  = (Network) Networks.get(i);
+                ITransformedNetwork transformedNetwork = transformationManager.getTransformedNetwork(n.getNetworkId());
+                String json = null;
+                if(transformedNetwork!=null){
+                    List<Link> links = transformedNetwork.getLinks();
+                    for(int itr=0; itr< links.size(); itr++){
+                        Node s = links.get(itr).getSubject();
+                        String url = s.getConceptId();
+                        String des = s.getDescription();
+                        String lbl = s.getLabel(); 
+                        if(top10Concepts.containsKey(url)){
+                            top10Concepts.put(url, top10Concepts.get(url)+1);
+                        }else{
+                            top10Concepts.put(url, 1);
+                            top10ConceptsDetails.put(url, new String[] {des, lbl});
+                        }
+                    }
+                    
+                }

```

Comment:
And the number of how many concepts should be shown in the statistics should not be hardcoded but configurable (for now through a system property, later we probably want to make that configurable by project).